net/pdnsd/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (C) 2019 Xiaobo Tian
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pdnsd
+PKG_VERSION:=1.2.9a
+PKG_RELEASE=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-par.tar.gz
+PKG_SOURCE_URL:=http://members.home.nl/p.a.rombouts/$(PKG_NAME)/releases/
+PKG_HASH:=bb5835d0caa8c4b31679d6fd6a1a090b71bdf70950db3b1d0cea9cf9cb7e2a7b
+
+PKG_MAINTAINER:=Xiaobo Tian <peterwillcn@gmail.com>
+PKG_LICENSE_FILES:=COPYING.GPLv3
```

Comment:
This file does not exist. Also, switch the order of PKG_LICENSE and PKG_LICENSE_FILES

Reply:
Add LICENSE and order